system-int-test/test/cmr/system_int_test/ingest/collection_progressive_update_test.clj

```diff
@@ -0,0 +1,89 @@
+              temporal-extents [(data-umm-cmn/temporal-extent {:beginning-date-time "2001-01-01T12:00:00Z"
+                                        :ending-date-time "2000-05-11T12:00:00Z"})]
+              data-dates [(umm-cmn/map->DateType {:Date (time-core/date-time 2032) :Type "DELETE"})]
+              ;; now that the previous concept containing validation errors, we don't need to pass options to mimic existing errors. 
+              response (data-core/ingest-umm-spec-collection "PROV1" (data-umm-c/collection
+                                                                      {:EntryTitle "ERS-2_LEVEL0"
+                                                                       :ShortName "ERS-2_L0"
+                                                                       :Version "1"
+                                                                       :AdditionalAttributes additional-attributes
+                                                                       :TemporalExtents temporal-extents
+                                                                       :DataDates data-dates}
+                                                                      ))
+              expected-warnings "After translating item to UMM-C the metadata had the following issue: {:existing-errors (\"[:TemporalExtents 0 :RangeDateTimes 0] [\\\"BeginningDateTime [2001-01-01T12:00:00.000Z] must be no later than EndingDateTime [2000-05-11T12:00:00.000Z]\\\"]\" \"[:AdditionalAttributes 0] [\\\"Parameter Range Begin is not allowed for type [STRING]\\\" \\\"Parameter Range End is not allowed for type [STRING]\\\"]\")}. [:TemporalExtents 0 :RangeDateTimes 0] BeginningDateTime [2001-01-01T12:00:00.000Z] must be no later than EndingDateTime [2000-05-11T12:00:00.000Z]. [:AdditionalAttributes 0] Parameter Range Begin is not allowed for type [STRING] Parameter Range End is not allowed for type [STRING]"
+              {:keys [errors warnings concept-id revision-id]} response]
+          (is (= concept-id (:concept-id coll)))
+          (is (= revision-id (+ 2 (:revision-id coll))))
+          (is (= nil errors))
+          (is (= expected-warnings warnings))))
+        
+        (testing "Progressive update collection failed case, introducing new data-dates errors"
+        (let [additional-attributes [(data-umm-cmn/additional-attribute {:Name "int" :DataType "STRING" :ParameterRangeBegin 1 :ParameterRangeEnd 10})]
+              temporal-extents [(data-umm-cmn/temporal-extent {:beginning-date-time "2001-01-01T12:00:00Z"
+                                        :ending-date-time "2000-05-11T12:00:00Z"})]
+              data-dates [(umm-cmn/map->DateType {:Date (time-core/date-time 2000) :Type "DELETE"})]
+              ;; now that the previous concept containing validation errors, we don't need to pass options to mimic existing errors.
+              response (try (data-core/ingest-umm-spec-collection "PROV1" (data-umm-c/collection
```

Comment:
The try/catch can be replaced with a 
```
(is (thrown-with-msg? 
       java.lang.Exception
       #"Ingest failed" 
       (data-core/ingest-umm-spec-collection ...
```

Reply:
Sorry I can't put the right message in to make this work.